csharp/src/Ice/Reference.cs

```diff
@@ -317,7 +326,7 @@ public bool Equals(Reference? other)
             {
                 return false;
             }
-            if (IsFixed != other.IsFixed)
+            if (IsRelative != other.IsRelative)
```

Comment:
Are you sure that it's correct to no longer compare IsFixed? Won't a well-known reference now potentially equal a fixed reference if they have the same attributes?

Reply:
You are right, this update was accidental. Fixed.